src/include/migraphx/matcher.hpp

```diff
@@ -239,7 +252,39 @@ struct any_matcher : any_matcher_base
 
 struct matcher_result
 {
-    std::unordered_map<std::string, instruction_ref> instructions;
+    struct instruction_container
```

Comment:
This change affects the CPU as well, so that's why the PR is failing on jenkins. https://github.com/ROCmSoftwarePlatform/AMDMIGraphX/blob/develop/src/targets/cpu/lowering.cpp#L355